.eslintrc.json

```diff
@@ -20,7 +20,11 @@
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/no-use-before-define": "off",
+    "@typescript-eslint/no-require-imports": "off",
     "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/no-empty-object-type": "off",
+    "@typescript-eslint/no-unused-expressions": "off",
+    "@typescript-eslint/prefer-as-const": "off",
```

Comment:
Why are those changes required?

Reply:
The linter fails otherwise, newer linter, the old version had audit issues

Reply:
I upgraded ESLint and improved the config. It should work now :)